dascore/core/coords.py

```
@@ -1561,6 +1569,150 @@ def reverse_sorted(self):
+            _raise_string_coord_error("range selection")
+        values = np.asarray([args], dtype=self.dtype)
+        return self._select_by_value_array(values)
+
+    def sort(self, reverse=False):
+        """Sort values lexicographically."""
+        inds = np.argsort(self.values)
+        if reverse:
+            inds = inds[::-1]
+        return self[inds], inds
+
+    def update_limits(self, min=None, max=None, step=None, **kwargs) -> Self:
```

Ruff (0.15.7)

[error] Function argument `min` is shadowing a Python builtin

(A002)